src/Twig/Extension/SonataAdminExtension.php

```diff
@@ -139,7 +146,7 @@ public function renderListElement(
     ) {
         $template = $this->getTemplate(
             $fieldDescription,
-            $fieldDescription->getAdmin()->getTemplate('base_list_field'),
+            $this->templateRegistry->getTemplate('base_list_field'),
```

Comment:
I think that's an issue right? How would you know if this is the correct template registry for that particular admin?

Reply:
I was doubting about that as well.

I'll change it.